### ISSUE TITLE
Accessibility: Platform Search Field does not announce the results popover content

### ISSUE THREAD

**Issue by kavya-b** [SAP/fundamental-ngx#5276]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
On navigating to 'Category' and pressing the button, or entering some characters in the search field when results popup in the popover, the results are not read out nor can be navigated.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
latest
#### If this is a bug, please provide steps for reproducing it.
1. Go to https://fundamental-ngx.netlify.app/#/platform/search-field 
2. Turn voiceover on, if using Mac, or any screen reader app.
3. Navigate using VoiceOver keyboard shortcuts(ctrl+option arrow keys) to the first example.
4. In the first example, start typing 'a'
5. Observe the popup comes with results
6. Similarly, in thhe next Ccategories example, navigate through VoiceOver over the Category button and Ctrl+Option+Space to click it.
<img width="1181" alt="Screenshot 2021-05-04 at 7 55 54 AM" src="https://user-images.githubusercontent.com/53509521/116959055-13bd4e00-acba-11eb-80bb-645b77af2b3b.png">
<img width="405" alt="Screenshot 2021-05-04 at 7 57 39 AM" src="https://user-images.githubusercontent.com/53509521/116959061-18820200-acba-11eb-83df-2bc9287bfb11.png">

Expected: Screen reader/Voiceover should read the existence of the popup and its contents.
Actual: Screen reader/Voiceover does not announce the existence of the popup nor reads its contents.

**Comment by puru-hk** [comment]
Tested working as expected.
